components/cdc/src/channel.rs

```diff
@@ -431,9 +431,8 @@ impl Drop for Drain {
         });
         block_on(&mut drain);
         let takes = start.saturating_elapsed();
-        if takes >= Duration::from_millis(200) {
-            warn!("drop Drain too slow"; "takes" => ?takes);
-        }
+        info!("drop Drain finished, free memory";"takes" => ?takes,
```

Comment:
The log doesn't show which connection the `Drain` belongs to. It's better to add this information.